### ISSUE TITLE
System errors not displayed in the error panel

### ISSUE THREAD

**Issue by jopasserat** [openmole/openmole#146]
A failed execution ended by an exception only displays the stack trace in the error panel:

![error](https://cloud.githubusercontent.com/assets/472083/14506273/d81bd3cc-01b4-11e6-9399-87e4600061f8.png)

while the actual error is reported in the console which launched OpenMOLE:
`ls: cannot access '/mnt/HCP100': No such file or directory`.

In a regular scala console, both are displayed together:
```
ls: cannot access '/tmpasjkc': No such file or directory
java.lang.RuntimeException: Nonzero exit value: 2
  at scala.sys.package$.error(package.scala:27)
  at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.slurp(ProcessBuilderImpl.scala:132)
  at scala.sys.process.ProcessBuilderImpl$AbstractBuilder.$bang$bang(ProcessBuilderImpl.scala:102)
  ... 33 elided
```

This might be related / to take into account with #107

**Comment by mathieuleclaire** [comment]
Do you still reproduce this kind of error with a fresh pull ?

**Comment by jopasserat** [comment]
Yes but with an improvement:
  - the standard error is still displayed in the console

```
[21313:21356:0511/224740:ERROR:nss_util.cc(839)] After loading Root Certs, loaded==false: NSS error code: -8018
Created new window in existing browser session.
May 11, 2016 10:47:47 PM org.openmole.ui.Application$$anonfun$run$1$$anonfun$apply$mcI$sp$2 apply
INFO: Server listening on port 41493.
ls: cannot access '/vol/vipdata/data/HCP100': No such file or directory

```

  - but the stacktrace goes under the _Failed_ panel

![toto](https://cloud.githubusercontent.com/assets/472083/15197759/2bc07ae2-17cb-11e6-94cc-67d38f216eca.png)

**Comment by jopasserat** [comment]
Below a minimalistic workflow to reproduce the problem:

```scala
import sys.process._

val dataDir     = "/leads/to/nowhere"

val subjectListTmp = (s"ls ${dataDir}" !!) split("\n") toList


EmptyTask()

```

**Comment by romainreuillon** [comment]
I think that it should be possible to collect the output stream of the execution of the OpenMOLE script.

**Comment by mathieuleclaire** [comment]
If we collect the output stream, we will also display the results of ToStringHook() ...

**Comment by jopasserat** [comment]
Is there a way to collect the error output instead? That's definitely where it's displayed, it's not part of stdout. It's just that everything ends up on the console in a terminal.

**Comment by romainreuillon** [comment]
Yes, I will fix it. The pb it that the streams are not collected during the script evaluation.

**Comment by romainreuillon** [comment]
Fixed by 70ae354d418c9f3ffb4c38e1d0097628924526db
